source/includes/php-mongodb-compatibility-table.rst

```diff
@@ -12,16 +13,21 @@
      - MongoDB 3.0
      - MongoDB 2.6
 
+   * - ext 1.8 + lib 1.7
```

Comment:
@ccho-mongodb: Does this violate your rule of not adding rows for unreleased driver versions?

If you mean to keep it, it should report compatibility for MongoDB 4.4 back through 3.0 (inclusive), as I had in my original PR (#653).

Reply:
Oops, forgot to paste. Just to confirm, will this version will be available when GA is announced?

Reply:
Yes. Both PHPC and PHPLIB will be released with server GA.